### PR TITLE
Adding post emberconf 2021 fastboot meetup link

### PR DESCRIPTION
**Pull request by kiwiupover**
After
image

### CODE REVIEW

**Review by suchitadoshi1987 — APPROVED** [review]

**Review by jaredgalanis — APPROVED** [review]

File: app/components/main-hero/meetup/template.hbs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<a
+  href="https://www.eventbrite.com/e/ember-fastboot-ssr-beyond-tickets-144468965565"
+  class="main-hero-meetup"
+  target="_blank"
+  rel="noopener">
+  <div class="content">
+    <p class="invite">
+      Joins us <span class="orange">&nbsp;Wed April 14th</span>
```
Comment (suchitadoshi1987) [inline app/components/main-hero/meetup/template.hbs]: Typo: Joins -> Join

Comment (suchitadoshi1987) [reply app/components/main-hero/meetup/template.hbs]: May be we should add in a time as well?

**Review by suchitadoshi1987 — DISMISSED** [review]

File: app/components/main-hero/meetup/template.hbs (hunk 1 above)
Comment (kiwiupover) [reply app/components/main-hero/meetup/template.hbs]: I have fixed the typo and added a time. I know it is not beautiful but it is functional.

**Review by mansona — APPROVED** [review]
Great work thanks!

**Pull request merged**